etcdutl/etcdutl/snapshot_command.go

```diff
@@ -62,6 +64,19 @@ The items in the lists are hash, revision, total keys, total size.
 	}
 }
 
+func newSnapshotHashKVCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hashkv <filename>",
+		Short: "Prints the KV history hash of a given file",
+		Long: `When --write-out is set to simple, this command prints out comma-separated status lists for each endpoint.
+The items in the lists are hash, hash revision, compact revision.
+`,
+		Run: SnapshotHashKVCommandFunc,
+	}
+	cmd.Flags().Int64Var(&hashKVRevision, "rev", 0, "maximum revision to hash (default: all revisions)")
```

Comment:
```suggestion
	cmd.Flags().Int64Var(&hashKVRevision, "rev", 0, "maximum revision to hash (default: latest revision)")
```

FYI. https://github.com/etcd-io/etcd/pull/15997